conf/deployment/aws/ipi_6az_rhcos_3m_6w.yaml

```diff
@@ -22,4 +22,4 @@ ENV_DATA:
   worker_replicas: 6
   master_replicas: 3
   # since us-east-1e doesn't support m5.4xlarge, we are using m4.2xlarge as worker_instance_type
-  worker_instance_type: 'm4.2xlarge'
+  worker_instance_type: 'm4.4xlarge'
```

Comment:
can you change the instance type in the above comment as well?

Reply:
ack